### ISSUE TITLE
py26 tests failing

### ISSUE THREAD

**Issue by jsha** [certbot/certbot#5269]
https://travis-ci.org/certbot/certbot/jobs/308555880#L5878-L5888

```
Collecting PyYAML==3.12 (from -c /home/travis/build/certbot/certbot/tools/pip_constraints.txt (line 47))
  Using cached PyYAML-3.12.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-2AB6Cr/PyYAML/setup.py", line 83, in <module>
        from wheel.bdist_wheel import bdist_wheel
      File "/home/travis/build/certbot/certbot/.tox/py26/lib/python2.6/site-packages/wheel/bdist_wheel.py", line 407
        ignore=lambda x, y: {'PKG-INFO', 'requires.txt', 'SOURCES.txt',
                                       ^
    SyntaxError: invalid syntax
```

**Comment by bmw** [comment]
I'm working on this (and other related testing issues). This will likely only affect Travis. While annoying, ignore this failure for now and I'll kick the builds when it's fixed.

**Comment by hugovk** [comment]
This can be closed, Python 2.6 dependencies have been pinned in https://github.com/certbot/certbot/pull/5281 and it's now passing.

(The sooner 2.6 can be [dropped](https://github.com/certbot/certbot/issues/5279) the better :)

**Comment by bmw** [comment]
Nice catch. Thanks @hugovk.
